bin/check-rds-events.rb

```diff
@@ -72,8 +84,18 @@ def maint_clusters
     rds = AWS::RDS::Client.new aws_config
 
     begin
-      # fetch all clusters identifiers
-      clusters = rds.describe_db_instances[:db_instances].map { |db| db[:db_instance_identifier] }
+      if !config[:db_instance_id].nil? && !config[:db_instance_id].empty?
+        db_instance = rds.describe_db_instances(db_instance_identifier: config[:db_instance_id])
+        if db_instance.nil? || db_instance.empty?
```

Comment:
yes, you can search for anything you like but that doesn't mean the rds instance actually exists. We should check the returned object to see if we actually got something back from amazon.